internal/batches/service/remote.go

```diff
@@ -0,0 +1,138 @@
+package service
+
+import (
+	"context"
+
+	"github.com/cockroachdb/errors"
+)
+
+var ErrServerSideBatchChangesUnsupported = errors.New("server side batch changes are not available on this Sourcegraph instance")
```

Comment:
I had a meta question about the user-facing way that refer to the feature known internally as SSBC, but it's not at all a specific concern about this PR, so I realized it's probably better suited to a team sync and will put it there instead. 🙂